repos/system_upgrade/el7toel8/actors/repositoriesmapping/libraries/library.py

```diff
@@ -28,10 +28,15 @@ def scan_repositories(path):
         next(data)  # skip header
 
         for row in data:
-            if len(row) != 6:
+            # skip empty lines and comments
+            if not row or row[0].startswith('#'):
```

Comment:
I guess we will not catch empty lines if they begin with white-space, however not sure whether we care that much as those will be caught later as invalid record...

Reply:
That is on purpose, I think it is better to keep the parsing as simple as possible, which servers two purposes: 1) less complex code, 2) motivation to keep the repomapping file clean.

Reply:
fair enough :)